### PR TITLE
fix(phone-number) - add validation for UK regions: Jersey and Guernsey

### PR DESCRIPTION
**Pull request by yasong-ui**
Jersey: +441534xxxxxx
Guernsey: +441481xxxxxx

https://jira-eng-gpk2.cisco.com/jira/browse/PROVISION-6513

### CODE REVIEW

File: src/lib/components/phoneNumber/phoneNumber.component.ts [hunk 1]
```diff
@@ -223,6 +223,10 @@ export class PhoneNumberCtrl implements ng.IComponentController {
         if (regionCode === 'us') {
           // Temporarily allow Bahamas numbers to be validated in the US region: PROVISION-5632
           return this.phoneUtil.isValidNumberForRegion(parsedNumber, 'us') || this.phoneUtil.isValidNumberForRegion(parsedNumber, 'bs');
+        } else if (regionCode === 'gb') {
+          return this.phoneUtil.isValidNumberForRegion(parsedNumber, 'gb') ||
+                  this.phoneUtil.isValidNumberForRegion(parsedNumber, 'je') ||
+                  this.phoneUtil.isValidNumberForRegion(parsedNumber, 'gg');
```
Comment (KHempel) [inline src/lib/components/phoneNumber/phoneNumber.component.ts:229]: Please leave a comment noting why this change is necessary

Comment (yasong-ui) [reply src/lib/components/phoneNumber/phoneNumber.component.ts]: Added.
